pennylane/optimize/gradient_descent.py

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 """Gradient descent optimizer"""
 
-import autograd
+from pennylane._grad import grad as get_gradient
```

Comment:
This is to avoid it clashing with an existing variable named `grad` I assume?

Reply:
Yes, that's right.